samples/Sentry.Samples.Console.ManualUserFeedback/Sentry.Samples.Console.ManualUserFeedback.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Please delete the custom Sample

Reply:
whoops sorry, I forgot that I also had to delete the folder.